tests/test_engine.py

```diff
@@ -15,11 +18,15 @@
 from .doubles import FakeThreadedReactor, ImmediateWorker
 
 
-def create_engine():
+def create_engine(**kwargs):
```

Comment:
Can you make this so that it generates _multiple_ test cases instead of just switching which one is being used? I'd like it if locally I got sqlite, and on Travis I got both sqlite _and_ postgres.